helpers/pluck.js

```diff
@@ -1,8 +1,30 @@
 'use strict';
 
+const helpers = require('handlebars-helpers');
+const object_helper = helpers.object();
```

Comment:
This helper is so simple that it's probably not worth bringing in this dependency.